docs/integrations/sources/notion.md

```diff
@@ -83,7 +83,8 @@ The connector is restricted by Notion [request limits](https://developers.notion
 ## Changelog
 
 | Version | Date       | Pull Request                                             | Subject                                                         |
-| :------ | :--------- | :------------------------------------------------------- | :-------------------------------------------------------------- |
+|:--------|:-----------|:---------------------------------------------------------|:----------------------------------------------------------------|
+| 1.0.0   | 2022-12-19 | [20639](https://github.com/airbytehq/airbyte/pull/20639) | Fix `Pages` stream schema                                       |
```

Comment:
You noted that this was a breaking change (1.0) - what should users expect to see that's changed from previous versions in their records?

Reply:
@davydov-d I think your intuition is correct. Even if SAT did not detect the backward incompatibility, this is a breaking change on the `page` stream. Could you please sync with @girarda and @jnr0790 to assess the impact of this breaking change and check if customer outreach is needed?
I'm under the impression you're fixing a nonworking stream, but it'd be worth double-checking that the stream is failing for all the users. It would help to spot the differences if you could share the output of `discover` in `0.1.10` vs `1.0.0`.

Reply:
Sure! I'm attaching the output for the `Pages` stream as files as it's pretty big

[0.1.10.txt](https://github.com/airbytehq/airbyte/files/10268166/0.1.10.txt)
[1.0.0.txt](https://github.com/airbytehq/airbyte/files/10268167/1.0.0.txt)

Reply:
The change you introduced is removing and adding properties.
The backward compatibility checks pass because:
- Removal of properties is considered backward compatible because downstream normalization is not destructive:  the previously created column for this property won't be deleted
- The addition of properties will not break an existing catalog but will trigger schema validation errors that are not failing syncs.

@evantahler @sherifnada  wdyt? Shall we make the backward compatibility test fail in this situation? 

I think that the change you introduced are not functionally breaking changes but will lead to very messy destination schema after normalization. 
I'd suggest you locally try a sync with `0.1.10` and then upgrade to 1.0.0:
- check if the sync succeeds
- check how the destination schema looks like

Reply:
I would think that any property removal should be a breaking change.  Regardless of if normalization removes the column or not, new records will be missing data that was previously a part of the record.

Reply:
I tracked this evolution in a issue: https://github.com/airbytehq/airbyte/issues/20754

Reply:
just to clarify - any action item here to be done? does it still make sense to run a sync locally to look at the destination schema regarding we have decided to treat it as an incompatible change or may I resolve this thread?

Reply:
This will be affecting all cloud users with a Notion connection correct?

Reply:
@jnr0790 this will affect those users with `Pages` stream enabled. I will post a list of them in the issue

Reply:
@davydov-d can you tag me and Erica on that comment cc @erica-airbyte